app/config.ts

```diff
@@ -0,0 +1,69 @@
+'use strict';
```

Comment:
You may want to keep this file a `js` file and a CommonJS module (i.e use require/module.exports instead of import/export) in case you need to pass some config to the client via the `publicRuntimeConfig` in the `next.config.js` (in which case this file wouldn't go through babel/tsc)